python/chronos/src/bigdl/chronos/forecaster/autoformer_forecaster.py

```diff
@@ -131,9 +135,14 @@ def __init__(self,
         self.model_config.update(self.loss_config)
         self.model_config.update(self.optim_config)
 
+        self.metrics = metrics
+
         self.distributed = distributed
-        self.seed = seed
         self.checkpoint_callback = True
+        # seed setting
+        if not isinstance(seed, Space):
+            from pytorch_lightning import seed_everything
+            seed_everything(seed=seed, workers=True)
```

Comment:
just a question here: do we need to set `workers=True` for other forecasters?(TCN, S2S, LSTM...)

Reply:
Set workers=True in [seed_everyhing](https://pytorch-lightning.readthedocs.io/en/stable/api/pytorch_lightning.utilities.seed.html?highlight=seed_everything#pytorch_lightning.utilities.seed.seed_everything)  will properly configure all dataloaders passed to the Trainer with a worker_init_fn.
I think this might be a more fixed and reproducible way and can set to other forecasters. But more exp of this parameter can be tested.

Reply:
sure, we will have some quick check on this param in future.